src/vcpkg/install.cpp

```diff
@@ -30,19 +30,7 @@ namespace
 {
     using namespace vcpkg;
     DECLARE_AND_REGISTER_MESSAGE(ResultsHeader, (), "Displayed before a list of installation results.", "RESULTS");
-    DECLARE_AND_REGISTER_MESSAGE(ResultsLine,
-                                 (msg::spec, msg::build_result, msg::elapsed),
-                                 "{Locked}",
-                                 "    {spec}: {build_result}: {elapsed}");
-
-    DECLARE_AND_REGISTER_MESSAGE(CmakeTargetsExcluded,
-                                 (msg::count),
-                                 "keep the indentation and the `#` mark",
-                                 "    # note: {count} targets were omitted.");
-    DECLARE_AND_REGISTER_MESSAGE(CmakeTargetLinkLibraries,
-                                 (msg::list),
-                                 "{Locked}",
-                                 "    target_link_libraries(main PRIVATE {list})");
+    DECLARE_AND_REGISTER_MESSAGE(CmakeTargetsExcluded, (msg::count), "", "note: {count} targets were omitted.");
```

Comment:
I know that this PR doesn't intend to make any functional changes, however we have gotten feedback from users that "omitted" was unclear.
```suggestion
    DECLARE_AND_REGISTER_MESSAGE(CmakeTargetsExcluded, (msg::count), "", "note: {count} additional targets are not displayed.");
```